openmdao/__init__.py

```diff
@@ -1 +1,3 @@
 __version__ = '2.4.0'
+
```

Comment:
Not sure how to feel about putting stuff in the __init__, since we have avoided doing it up to this point. What about someting like an opendmao.settings?

Reply:
I think putting constants in __init__ is fine.  The problems come when you import things in __init__.